scripts/skills/actives/rf_kata_step_skill.nut

```diff
@@ -37,23 +37,48 @@ this.rf_kata_step_skill <- ::inherit("scripts/skills/skill", {
 		if (this.getContainer().getActor().isPlacedOnMap())
 			return this.skill.getCostString();
 
-		local ret = "Costs " + (this.m.APCostModifier == 0 ? "+0" : ::MSU.Text.colorizeValue(this.m.APCostModifier, {AddSign = true, InvertColor = true})) + " [Action Points|Concept.ActionPoints] and builds ";
-		ret += (this.m.FatigueCostModifier == 0 ? "+0" : ::MSU.Text.colorizeValue(this.m.FatigueCostModifier, {AddSign = true, InvertColor = true})) + " [Fatigue|Concept.Fatigue] compared to the movement costs of the starting tile";
-		return ::Reforged.Mod.Tooltips.parseString(ret);
+		return "Cost is equal to the movement cost of the starting tile";
```

Comment:
I agree with all the changes except the cost string and AP/Fatigue cost display. I also have a significant refactor of this skill as part of my `rework-sword-perks` branch which already incorporates some of the changes that are being done in this PR. I'd say rebase this on top of that or I can just a the implement the changes that I like from this into that one.

Reply:
You implement the changes you like into your PR